frontend/src/components/website/adding_master/StepsProgressBar.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+    <VContainer>
+        <VLayout class="progress-panel">
+            <VFlex
+                ma-1
+                xs4
+                sm4
+                md4
+                class="progress-step"
+                :class="{'active': stepNumber == 1,'filled': this.$parent.currentStepNumber >= 1}"
+            >
+                <div class="font-weight-bold step-name">Add name</div>
+            </VFlex>
+            <VFlex
+                ma-1
+                xs4
+                sm4
+                md4
+                class="progress-step"
+                :class="{'active': stepNumber == 2,'filled': this.$parent.currentStepNumber >= 2}"
+            >
+                <div class="font-weight-bold step-name">Add website</div>
+            </VFlex>
+            <VFlex
+                ma-1
+                xs4
+                sm4
+                md4
+                class="progress-step"
+                :class="{'active': stepNumber == 3,'filled': this.$parent.currentStepNumber > 2}"
+            >
+                <div class="font-weight-bold step-name"
+                >
+                    Tracking info
+                </div>
+            </VFlex>
+        </VLayout>
+    </VContainer>
+</template>
+
+<script>
+    export default {
+        name: 'StepsProgressBar',
+        computed: {
+            stepNumber () {
+                return this.$parent.currentStepNumber;
```

Comment:
avoid using parent reference, component shouldn't know anything about it's parent, since it breaks single direction flow